tests/queries/0_stateless/02028_add_default_database_for_alterquery_on_cluster.sql

```diff
@@ -0,0 +1,17 @@
+-- Tags: distributed, no-parallel
```

Comment:
Seems like it should fix the problem.
```suggestion
-- Tags: distributed, no-parallel, no-replicated-database
-- Tag no-replicated-database: ON CLUSTER is not allowed

```